build-libs/redirects.js

```diff
@@ -122,7 +119,6 @@ async function buildProductRedirects() {
 			getRedirectsFromContentRepo('boundary'),
 			getRedirectsFromContentRepo('nomad'),
 			getRedirectsFromContentRepo('vault'),
-			getRedirectsFromContentRepo('waypoint'),
```

Comment:
Not strictly related to integrations, but this is unnecessary as all Waypoint docs redirect to https://github.com/hashicorp/waypoint/tree/main/website/content/docs